src/index.js

```diff
@@ -8,7 +8,7 @@ import styles from './styles.css';
 // =============================================================================
 function docsifyCopyCode(hook, vm) {
     hook.doneEach(function() {
-        const codeBlocks = Array.apply(null, document.querySelectorAll('pre[v-pre]'));
+        const codeBlocks = Array.apply(null, document.querySelectorAll('pre[data-lang]'));
```

Comment:
Is the `data-lang` attribute always present? We should potentially think about making this selector dynamic in the plugin config so the user can decide which elements to find in the DOM.

Reply:
Yep. Even on code blocks that don't have a language specified. Here's what the output looks like:

```html
<pre v-pre="" data-lang="">
  <code class="lang-">This is some code</code>
</pre>
```

Notice the `v-pre` attribute. This is the attribute that gets removed by when the second Vue instance re-renders the content area. After the content has been re-rendered, the above markup looks like this:

```html
<pre data-lang="">
  <code class="lang-">This is some code</code>
</pre>
```

Weird, but that's the deal. Hence the selector change.

I'm not sure making the selector configurable is necessary though, unless you want folks to be able to copy content other than code blocks.

Reply:
I also did not think making the selector configurable would be necessary but had someone reach out to me a while ago on Twitter asking about it, but might have been because this issue caused problems for them. Let's leave that for now.